### PR TITLE
Add typings for setName

### PR DESCRIPTION
**Pull request by mAAdhaTTah**

### CODE REVIEW

File: kefir.js.flow [hunk 1]
```diff
@@ -46,6 +46,8 @@ declare class Observable<+V,+E=*> {
   offLog(name?: string): this;
   spy(name?: string): this;
   offSpy(name?: string): this;
+  setName(name?: string): this;
```
Comment (rpominov) [inline kefir.js.flow]: `name` is not optional here.

File: kefir.js.flow [hunk 2]
```diff
@@ -46,6 +46,8 @@ declare class Observable<+V,+E=*> {
...
+  setName(Observable<V, E>, name?: string): this;
```
Comment (rpominov) [inline kefir.js.flow]: I think this should be `setName(Observable<any, any>, name: string): this` i.e., say an `Observable` of `number` can be created from an `Observable` of another type, not necessarily `number`. And  `name` also not optional.

**Comment by mAAdhaTTah** [comment]
Updated.

**Comment by rpominov** [comment]
Thanks! I'll release a version with this change soon.

**Comment by rpominov** [comment]
`3.7.1` is out! Sorry for the delay. And thanks again for the PR!

**Pull request merged**